src/o365/graph/commands/teams/teams-channel-list.ts

```diff
@@ -0,0 +1,105 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import Utils from '../../../../Utils';
+import { Channel } from './Channel';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  groupId: string;
+}
+
+class GraphTeamsChannelListCommand extends GraphItemsListCommand<Channel>{
+  public get name(): string {
+    return `${commands.TEAMS_CHANNEL_LIST}`;
+  }
+  public get description(): string {
+    return 'List the channels in a specified Microsoft Teams team';
+  }
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: () => void): void {
+    const endpoint: string = `${auth.service.resource}/beta/teams/${args.options.groupId}/channels`; this
+      .getAllItems(endpoint, cmd, true)
+      .then((): void => {
+        if (args.options.output === 'json') {
+          cmd.log(this.items);
+        }
+        else {
+          cmd.log(this.items.map(m => {
+            return {
```

Comment:
In the default output, we're showing too many properties. In a regular console window, the output is not readable:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/11164679/53297211-45edde00-381b-11e9-86d3-0ba825e0567d.png">

For the default output, let's limit it to just the displayName and the id. We'll show all available properties in the JSON output.